app/src/components/AddCredentialButton.tsx

```diff
@@ -1,20 +1,15 @@
-import React, {useCallback} from 'react'
+import React, { useCallback } from 'react'
 import { useTranslation } from 'react-i18next'
-import { StyleSheet, TouchableOpacity } from 'react-native'
+import { StyleSheet, TouchableOpacity, DeviceEventEmitter } from 'react-native'
```

Comment:
I'm having trouble finding docs for `DeviceEventEmitter`. Could you link where you found info about this API? Also, does it support both Android and iOS in 0.66.x?

Reply:
it should support both since it's built in to react-native. The documentation does seem to be lacking, but I figured it's best to use something that is already in react native rather than importing a 3rd party library. Here's some documentation I found, the best documentation seems to be for a similar module `NativeEventEmitter`:
https://reactnative.dev/docs/native-modules-android#sending-events-to-javascript
https://scarcoco.github.io/react-native/docs/native-modules-android#sending-events-to-javascript